src/test/java/com/onto/javacourse/intro/summinglargenumbers/SummingLargeNumbersTest.java

```diff
@@ -0,0 +1,92 @@
+package com.onto.javacourse.intro.summinglargenumbers;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class SummingLargeNumbersTest {
```

Comment:
What about "100001 0" - I think this might fail?  Or "99999 1?

Reply:
i will add tests to show that they work with the current code